css-basics/css-basics_combined-users/css/styles.css

```diff
@@ -46,24 +46,51 @@ li {
   background-color: white;
 }
 
+aside .user {
+  position: relative;
+  border: 2px solid lightslategray;
+  border-radius: 4px;
+  box-shadow: 4px 4px 0 lightslategray;
+  padding: 12px;
+  max-width: 250px;
+  margin: 0;
+  background-color: white;
+}
+
 .name {
-  font-size: 18px;
+  font-size: 16px;
```

Comment:
Here it was necessary to change the styles of .name only those that are inside the aside (add aside .name{...}), and the styles of .name should not be touched.